instrumentation/aws_lambda/lib/opentelemetry/instrumentation/aws_lambda/instrumentation.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module AwsLambda
+      # Instrumentation class that detects and installs the AwsLambda instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        install do |_config|
+          require_dependencies
+        end
+
+        present do
+          # maybe check if ORIG_HANDLER or _HANLDER exist
+          true
+        end
+
+        compatible do
+          true
+        end
```

Comment:
Are you aware of any versions of the gem that aren't compatible with the instrumentation? Are there any versions that have been EOL'd by AWS that we explicitly don't want to support?

Reply:
I think aws instrumentation doesn't rely on any external gem that require prepend on methods, so I think it should always be compatible; but please correct me if I miss something.

Reply:
You're correct, thanks for clarifying!